### PR TITLE
Brand Index - Create - Search

### PR DESCRIPTION
**Pull request by phanlyhuynh**
Index, create, search Brand

### CODE REVIEW

**Comment by phanlyhuynh** [comment]
ready

File: resources/views/admin/brand/index.blade.php [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+                                    @foreach($brands as $brand)
+                                        <tr role="row" class="odd">
+                                            <td class="sorting_1">
+                                                <div class="checkbox">
+                                                    <input id="selectable2" type="checkbox">
+                                                    <label for="selectable2"></label>
+                                                </div>
+                                            </td>
+                                            <td>{{ $brand->name }}</td>
+                                            @if($brand->status==0)
```
Comment (BunnyPi04) [inline resources/views/admin/brand/index.blade.php]: convention

File: resources/views/admin/master.blade.php [hunk 2]
```diff
@@ -84,6 +84,6 @@
 <script src="{{ asset('vendor/selectize/dist/js/standalone/selectize.min.js') }}"></script>
 <script src="{{ asset('vendor/summernote/dist/summernote-bs4.min.js') }}"></script>
 <script src="{{ asset('vendor/bootstrap-datepicker/dist/js/bootstrap-datepicker.js') }}"></script>
-{{--<script src="{{ asset('js/tables/data-table.js') }} "></script>--}}
+<script src="{{ asset('js/forms/form-elements.js')}}"></script>
```
Comment (BunnyPi04) [inline resources/views/admin/master.blade.php]: convention

**Comment by BunnyPi04** [comment]
commented

**Comment by phanlyhuynh** [comment]
ready

File: app/Http/Controllers/Admin/BrandController.php [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+    public function index(Request $request)
+    {
+        $brands = Brand::where('id', '>', 0);
+        $keyword = $request->keyword;
+        if ($request->has('keyword'))
+        {
+            $brands = $brands->where('name', 'like', '%' . $keyword . '%');
+        }
+        $brands = $brands->orderBy('id', 'ASC')->get();
```
Comment (BunnyPi04) [inline app/Http/Controllers/Admin/BrandController.php]: nghiên cứu sử dụng scope nhé

**Comment by BunnyPi04** [comment]
commented

**Comment by phanlyhuynh** [comment]
ready

**Pull request merged**